h2gis-functions/src/main/java/org/h2gis/functions/factory/H2GISSimpleDBFactory.java

```diff
@@ -0,0 +1,272 @@
+/**
+ * H2GIS is a library that brings spatial support to the H2 Database Engine
+ * <http://www.h2database.com>. H2GIS is developed by CNRS
+ * <http://www.cnrs.fr/>.
+ *
+ * This code is part of the H2GIS project. H2GIS is free software; 
+ * you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation;
+ * version 3.0 of the License.
+ *
+ * H2GIS is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License
+ * for more details <http://www.gnu.org/licenses/>.
+ *
+ *
+ * For more information, please consult: <http://www.h2gis.org/>
+ * or contact directly: info_at_h2gis.org
+ */
+
+package org.h2gis.functions.factory;
+
+import org.h2.engine.Constants;
+import org.h2.jdbcx.JdbcDataSource;
+import org.h2.util.OsgiDataSourceFactory;
+
+import javax.sql.DataSource;
+import java.io.File;
+import java.net.URI;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+/**
+ * Used to create quickly a database on unit tests. Should be used for test purpose ONLY
+ *
+ * @author Nicolas Fortin
+ * @author Erwan Bocher
+ * @author Sylvain Palominos
+ */
+public class H2GISSimpleDBFactory {
```

Comment:
Why name it H2GISSimpleDBFactory instead of H2GISDBFactory ?

Reply:
Done